package.json

```diff
@@ -16,7 +16,7 @@
     "sinon": "^1.17.6"
   },
   "scripts": {
-    "test": "./node_modules/mocha/bin/mocha tests/js/tests/*.test.js"
+    "test": "./node_modules/mocha/bin/mocha tests/js/tests/*.test.js --timeout 30000"
```

Comment:
@dynasource Seems like it's Vagrant problem. This error appeared after adding jsdom, so with empty test or tests without jsdom default timeout (2000ms) is not exceeded.

As for specific value (15000ms) - I tried to increase it gradually till the error is gone. But sometimes even `15000ms` was not enough so I increased it even more. I noticed that error generally happens on the first run of `npm test` then it's gone (I mean the case when it's set 15000ms, 2000ms is always not enough for me on Vagrant).

Let's try to remove timeout completely and see if default will be enough for Travis. We can increase it when it will be necessary. a Also adding note to the docs won't be excess.